Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h

```diff
@@ -44,6 +44,7 @@ enum class SDKAlignedBehavior {
     DefaultsToPassiveWheelListenersOnDocument,
     DisallowsSettingAnyXHRHeaderFromFileURLs,
     DoesNotDrainTheMicrotaskQueueWhenCallingObjC,
+    DoNotParseStringEndingWithFullStopAsFloatingPointNumber,
```

Comment:
I admit there’s not a lot of consistency here, but when I renamed them all I was shooting for “what is the behavior of X SDK” (so “Does Not” prefix), not “what should I do” (“Do Not” prefix). I’m sure there’s a grammatical word for this :)

Reply:
Makes sense! I'll rename this to something along the lines of, `TreatStringEndingWithFullStopAsInvalidFloatingPointNumber`.